fe/fe-core/src/main/java/com/starrocks/analysis/InlineViewRef.java

```diff
@@ -331,4 +331,28 @@ public String tableRefToSql() {
 
         return sb.toString();
     }
+
+    @Override
+    public String tableRefToDigest() {
+        // Enclose the alias in quotes if Hive cannot parse it without quotes.
+        // This is needed for view compatibility between Impala and Hive.
+        String aliasSql = null;
+        String alias = getExplicitAlias();
+        if (alias != null) {
+            aliasSql = ToSqlUtils.getIdentSql(alias);
+        }
+        if (view != null) {
+            // TODO(zc):
+            // return view_.toSql() + (aliasSql == null ? "" : " " + aliasSql);
```

Comment:
remove

Reply:
All codes in this pr don't exist in main branch